### PR TITLE
Consume vscode 1.1.10 with Insiders build workaround

### PR DESCRIPTION
**Pull request by guw**
### What does this PR do?
Updates the vscode devDependencies to 1.1.10 in order to be able to run tests against VS Code Insiders build on Linux. 

### What issues does this PR fix or reference?
Adress build issues on Linux with current VS Code app.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "extends": ["prettier"],
+  "env": {
+    "es6": true,
+    "node": true
+  }
+}
```
Comment (guw) [inline .eslintrc.json]: This file has been created to support prettier formatting rules when processing JS files with eslint (see scripts directory).

File: .travis.yml [hunk 2]
```diff
@@ -92,3 +91,7 @@ script:
 
 after_script:
   - rm $SFDX_CI_KEY_LOCATION
+
+after_failure:
+  - find ./ -iname '*.log' -type f -print -exec cat '{}' \;
+  - find /home/travis/.npm/_logs/ -iname '*.log' -type f -print -exec cat '{}' \;
```
Comment (guw) [inline .travis.yml]: cat log files to build log on failures

**Comment by guw** [comment]
@DatGuyJonathan This makes the tests pass again on Windows as well as Linux.

File: packages/salesforcedx-vscode-apex-debugger/package.json [hunk 3]
```diff
@@ -169,5 +169,10 @@
         "title": "%exception_breakpoint_command_text%"
       }
     ]
+  },
+  "__metadata": {
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-vscode-apex-debugger/package.json]: @guw What are these for?

Comment (guw) [reply packages/salesforcedx-vscode-apex-debugger/package.json]: Hmm, very weird. I haven't noticed. Could that be a leftover from build?

Comment (guw) [reply packages/salesforcedx-vscode-apex-debugger/package.json]: @DatGuyJonathan remove it

**Pull request merged**